docs/reference/cli.md

```diff
@@ -115,18 +94,17 @@ quickwit index create
 
 *Examples*
 
-*Create a new index with metastore in the local `metastore` folder.*
+*Create a new index with the default local metastore.*
 ```bash
-quickwit index create --index-config-uri ./wikipedia_index_config.yaml  --config=quickwit.yaml
+quickwit index create --index wikipedia --index-config wikipedia_index_config.yaml  --config=./config/quickwit.yaml
 ```
 
 ### index ingest
 
 Indexes a dataset consisting of newline-delimited JSON objects located at `input-path` or read from *stdin*. 
-The data is appended to the target index specified by `index` unless `overwrite` is passed. `input-path` can be a file or another command output piped into stdin. 
-Currently, only local datasets are supported. 
-By default, quickwit's indexer will work with a heap of 2 GiB of memory, but this can be set with the `heap-size` option in the indexing_settings.resources section in the config. 
-This does not directly reflect the overall memory usage of `quickwit index ingest`, but doubling this value should give a fair approximation.
```

Comment:
this is useful information, or is it outdated?

Reply:
I prefer to have the details on the index config page to avoid duplication. Though the writing is not really good here, I will improve it.